### PR TITLE
trialing filterrific

### PR DESCRIPTION
**Pull request by nalnaji**
Darren, I'm not exactly sure if this will work.. But there's only one way to find out! 
Reviewer: @DarryQueen

### CODE REVIEW

File: app/controllers/video_controller.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class VideoController < ApplicationController
+  def index
+    @filterrific = initialize_filterrific(
```
Comment (DarryQueen) [inline app/controllers/video_controller.rb]: Does this variable ever get referenced by views? If not, no need to set it as instance variable.

File: config/routes.rb [hunk 2]
```diff
@@ -6,7 +6,7 @@
   # Set home pages:
   root :to => 'home#index'
   get 'home' => 'home#dashboard', :as => :dashboard
-
+  get 'videos' => 'videos#index'
```
Comment (DarryQueen) [inline config/routes.rb]: You don't even have an index template... This is guaranteed not to work.

Comment (nalnaji) [inline config/routes.rb]: Yeah, I know! I need yours! You said you were building it, right?

Comment (DarryQueen) [inline config/routes.rb]: I built a basic AJAX system. I'll push when I've finished making it a teeny bit prettier.

**Pull request closed without merging**